File: Producto2/juego.f90
program juego
  write(*,*) 'Hola! Trataré de adivinar un número.'
  write(*,*) 'Piensa un número entre 1 y 10.'
  call sleep(5)
  write(*,*) 'Ahora multiplícalo por 9.'
  call sleep(5)
  write(*,*) 'Si el múmero tiene 2 dígitos, súmalos entre si: Ej. 36 -> 3=6+9. Si tu número tiene un solo dígito, súmale 0.'
  call sleep(5)
  write (*,*) 'Al número resultante súmale 4.'
  call sleep(8)
  write (*,*) 'Muy bien. El resultado es 13 :)'
end program juego

File: Producto2/juego.c
/* Juego en C, Ansi-Style */
#include <stdio.h>
#include <stdlib.h>
#include <unistd.h>

int main(void)
{
puts("Hola! Trataré de adivinar un número.");
puts("Piensa un número entre 1 y 10.");
sleep(5);
puts("Ahora multiplícalo por 9.");
sleep(5);
puts("Si el número tiene 2 dígitos, súmalos entre si: Ej. 36 -> 3+6=9. Si tu número tiene un solo dígito, súmale 0.");
sleep(5);
puts("Al número resultante súmale 4.");
sleep(8);
puts("Muy bien. El resultado es 13 :)");
return(EXIT_SUCCESS);
}



